shared/teams/team/rows/channel-row/channel.tsx

```diff
@@ -30,7 +30,7 @@ const ChannelRow = (props: ChannelRowProps) => {
     dispatch(TeamsGen.createSetChannelSelected({channel: channel.channelname, selected, teamID}))
   }
   const onEditChannel = () =>
-    dispatch(nav.safeNavigateAppendPayload({path: [{props, selected: 'chatEditChannel'}]}))
+    dispatch(nav.safeNavigateAppendPayload({path: [{props, selected: 'teamEditChannel'}]}))
```

Comment:
Designs have this button going to the edit channel modal + clicking the list item body going to the channel page.

Reply:
Doesn't clicking the list item body open the multi-select?

Reply:
https://zpl.io/bzXN5z4
https://zpl.io/2Z1jwow
https://zpl.io/brX8yOL
https://zpl.io/a7BOk3K

Reply:
See note here https://zpl.io/VQ7wQ0n

Reply:
I see--this is different from my previous conversations with Design. It might need some bigger changes to `ListItem2`.

Reply:
Actually nvmind, it should be pretty easy.